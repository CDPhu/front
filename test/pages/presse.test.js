import renderer from 'react-test-renderer';
import Index, { getStaticProps } from '../../pages/presse';

jest.mock('../../components/head');
jest.mock('../../components/blog-index');
jest.mock('../../components/navbar');
jest.mock('../../components/footer');
jest.mock('../../components/newsletter');
jest.mock('../../lib/directus');

const renderTree = tree => renderer.create(tree);
describe('<Index>', () => {
  it('should render component', () => {
    expect(renderTree(<Index />).toJSON()).toMatchSnapshot();
  });
  
});


describe('getStaticProps', () => {
  it('should expose a function', () => {
		expect(getStaticProps).toBeDefined();
	});
  
  it('getStaticProps should return expected output', async () => {
    const retValue = await getStaticProps();
    expect(retValue).toBeTruthy();
  });
});

