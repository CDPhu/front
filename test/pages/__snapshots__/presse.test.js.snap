// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index> should render component 1`] = `
<div
  className="relative pt-24 pb-24 px-4 sm:px-6 md:pt-28"
/>
`;
